runtime/browser/xwalk_runner.cc

```diff
@@ -144,6 +144,8 @@ void XWalkRunner::OnRenderProcessWillLaunch(content::RenderProcessHost* host) {
   std::vector<extensions::XWalkExtension*> ui_thread_extensions;
   std::vector<extensions::XWalkExtension*> extension_thread_extensions;
 
+// Disable bundled extensions for Linux desktop environment.
+#if defined(OS_TIZEN) || !defined(OS_LINUX)
```

Comment:
Thanks for mention this, I will add a new XWalkRunnerLinux to handle the derivation.